format/index.md

```diff
@@ -355,6 +360,10 @@ responses are called "compound documents".
 In a compound document, all included resources **MUST** be represented as an
 array of resource objects in a top-level `"included"` member.
 
+In a compound document, any resource which references other resources included 
+into the same document (either in top-level `"data"` or `"included"` members)
+**MUST** include references to those resources in its own `"links"` member.
+
```

Comment:
Again, I don't understand the purpose of this paragraph. How can a resource reference other resources apart from listing them under `links`?